blueprints/BLUEPRINT_TEMPLATE.md

```diff
@@ -0,0 +1,190 @@
+What fields are required or optional?
+What are the defaults for optional fields?
+What are the validation constraints?
+-->
+
+#### Client Impact
+<!--
+*How* will this impact Traffic Ops REST API clients (Go, Python, Java)?
+
+If new endpoints are required, will corresponding client methods be added?
+-->
+
+#### Data Model / Database Impact
+<!--
+*How* will this impact the Traffic Ops data model?
+*How* will this impact the Traffic Ops database schema?
+
+What changes to the lib/go-tc structs will be required?
+What new tables and columns will be required?
+How will existing tables and columns be changed?
+What are the column data types and modifiers?
+What are the FK references and constraints?
+-->
+
+### ORT Impact
+<!--
+*How* will this impact ORT?
+-->
+
+### Traffic Monitor Impact
+<!--
+*How* will this impact Traffic Monitor?
+
+Will new profile parameters be required?
+-->
+
+### Traffic Router Impact
+<!--
+*How* will this impact Traffic Router?
+
+Will new profile parameters be required?
+How will the CRConfig be changed?
+How will changes in Traffic Ops data be reflected in the CRConfig?
+Will Traffic Router remain backwards-compatible with old CRConfigs?
+Will old Traffic Routers remain forwards-compatible with new CRConfigs?
+-->
+
+### Traffic Stats Impact
+<!--
+*How* will this impact Traffic Stats?
+-->
+
+### Traffic Vault Impact
+<!--
+*How* will this impact Traffic Vault?
+
+Will there be any new data stored in or removed from Riak?
+Will there be any changes to the Riak requests and responses?
+-->
+
+### Documentation Impact
+<!--
+*How* will this impact the documentation?
+
+What new documentation will be required?
+What existing documentation will need to be updated?
+-->
+
+### Testing Impact
+<!--
+*How* will this impact testing?
+
+What is the high-level test plan?
+How should this be tested?
+Can this be tested within the existing test frameworks?
+How should the existing frameworks be enhanced in order to test this properly?
+-->
+
+### Performance Impact
+<!--
+*How* will this impact performance?
+
+Are the changes expected to improve performance in any way?
+Is there anything particularly CPU, network, or storage-intensive to be aware of?
+What are the known bottlenecks to be aware of that may need to be addressed?
+-->
+
+### Security Impact
+<!--
+*How* will this impact overall security?
+
+Are there any security risks to be aware of?
+What privilege level is required for these changes?
+Do these changes increase the attack surface (e.g. new untrusted input)?
+How will untrusted input be validated?
+If these changes are used maliciously or improperly, what could go wrong?
+Will these changes adhere to multi-tenancy?
+Will data be protected in transit (e.g. via HTTPS or TLS)?
+Will these changes require sensitive data that should be encrypted at rest?
+Will these changes require handling of any secrets?
+Will new SQL queries properly use parameter binding?
+-->
+
+### Upgrade Impact
```

Comment:
Instead of trying to lump these big topics (testing, operations, upgrades, etc) at the end, I think (personal preference) we should be asking those questions for every component with a simple overarching summary if needed.